extensions/http-curl/tests/C2VerifyHeartbeatAndStop.cpp

```diff
@@ -55,7 +40,7 @@ class VerifyC2Heartbeat : public VerifyC2Base {
 
   void runAssertions() override {
     using org::apache::nifi::minifi::utils::verifyLogLinePresenceInPollTime;
-    assert(verifyLogLinePresenceInPollTime(std::chrono::milliseconds(wait_time_),
+    assert(verifyLogLinePresenceInPollTime(std::chrono::milliseconds(120000),
```

Comment:
what is the reason behind this hardcoded wait time?

Reply:
Thanks for catching this, I only set this while testing something and I forgot about it. Fixed in a064ba2a3bab0dbe4ded393a4d52dd5315c3f976